enterprise/app/api_keys/api_keys.tsx

```diff
@@ -390,8 +422,20 @@ export default class ApiKeysComponent extends React.Component<ApiKeysComponentPr
   }
 }
 
-function isReadOnly<T extends ApiKeyFields>(apiKey?: T | null) {
+function hasCapability<T extends ApiKeyFields>(apiKey: T | null, capability: api_key.ApiKey.Capability) {
   return apiKey?.capability
```

Comment:
nit: I think this could be simplified to `return Boolean(apiKey?.capability?.some(...))`

Reply:
Done.